dev/Repeater/SelectionModel.cpp

```diff
@@ -190,7 +190,7 @@ winrt::IVectorView<winrt::IInspectable> SelectionModel::SelectedItems()
                     const unsigned int currentCount = node->SelectedCount();
                     if (index >= currentIndex && index < currentIndex + currentCount)
                     {
-                        int targetIndex = node->SelectedIndices().at(index - currentIndex);
+                        cnst int targetIndex = node->SelectedIndices().at(index - currentIndex);
```

Comment:
Typo.

Reply:
thanks! I had this fixed locally but forgot to push :-(. Pushed now...